src/utilities/toast-manager.test.ts

```diff
@@ -1,3 +1,92 @@
+import faker from "faker";
+import { ToastManager } from "./toast-manager";
+
 describe("ToastManager", () => {
-    test.skip("TODO: Backfill tests for issue https://github.com/AndcultureCode/AndcultureCode.JavaScript.React.Components/issues/34", () => {});
+    test("when error toast created, returns toastId", () => {
+        // Arrange
+        const testToastId = faker.random.number(99);
```

Comment:
Thought: Being as this is the value we're expecting to be returned from the function under test, this would be a good candidate to be named `expected` 👍